src/test/java/com/nimbusds/jose/crypto/ECDHCryptoTest.java

```diff
@@ -361,6 +361,32 @@ public void testCycle_ECDH_ES_Curve_P256_DecryptWithJose4j()
 		assertEquals("Hello world!", jwe.getPlaintextString());
 	}
 
+	public void testCycle_ECDH_ES_Curve_P256_XC20P()
```

Comment:
Can we have tests for other curves and for ECDH1-PU? Probably we can extend Cycle tests there to include XC20P?

Reply:
I did